src/pages/Liquidated.tsx

```diff
@@ -29,50 +22,31 @@ export function Liquidated() {
   };
 
   //  Queries for graph
-  const { data: tokenNamesData } = useSWR<AxiosResponse, AxiosError>(LIQUIDATION_GRAPH_TOKENS_QUERY, subQueryFetcher);
-  const tokenNamesResponse: TokenNamesResponse = tokenNamesData?.data.data;
-  const tokenNames = tokenNamesResponse?.vaultManagerMetrics.nodes.map((node) => node.liquidatingCollateralBrand) || [];
-
-  const { data: dailyMetricsData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
-    LIQUIDATION_DAILY_METRICS_QUERY(tokenNames),
-    subQueryFetcher,
+  const { data: vaultStateDailyData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
+    VAULT_STATE_DAILIES_QUERY,
+    (query: string) => subQueryFetcher(query, SUBQUERY_STAGING_URL),
   );
-  const dailyMetricsResponse: LiquidationMetricsDailyResponse = dailyMetricsData?.data.data;
+  const vaultStateDailyResponse: VaultStateDailyResponse = vaultStateDailyData?.data.data;
```

Comment:
Optional Chaining: `vaultStateDailyData?.data?.data`

Reply:
if vaultStateDailyData exists then `vaultStateDailyData.data.data` will always exist

Reply:
But what if `vaultStateDailyData` is an empty object or does not contain `data` property? I remember when queries where timing out, the `data` property was not there.